docs/features/pointing_device.md

```diff
@@ -927,3 +928,827 @@ In general the following two functions must be implemented in appropriate locati
+    _PM_RGB_MSP,  // eighth mode map  [index 7]
+    _PM_GAME      // ninth mode map   [index 8]
+};
+
+// Note POINTING_MODES_NUM_DIRECTIONS is required
+const uint16_t PROGMEM pointing_modes_maps[][POINTING_MODES_NUM_DIRECTIONS] = {
+    [_PM_BROW] = {
+                     KC_NO,
+        C(S(KC_TAB)),       C(KC_TAB),
+                     KC_NO
+    },
+    [_PM_APP] = {
+                     KC_NO,
+        A(S(KC_TAB)),           A(KC_TAB),
+                     KC_NO
+    },
+    [_PM_CARET] = {
+                    KC_UP,
+        KC_LEFT,                KC_RIGHT,
+                    KC_DOWN
+    },
+    [_PM_VOLUME] = {
+                    KC_VOLU,
+        KC_NO,                  KC_NO,
+                    KC_VOLD
+    },
+    [_PM_HISTORY] = {
+                    KC_NO,
+        C(KC_Z),                C(KC_Y),
+                    KC_NO
+    },
+    [_PM_RGB_MB] = {
+                    RGB_VAI,
+        RGB_RMOD,               RGB_MOD,
+                    RGB_VAD
+    },
+    [_PM_RGB_HS] = {
+                    RGB_HUI,
+        RGB_SAD,                RGB_SAI,
+                    RGB_HUD
+    },
+    [_PM_RGB_MSP] = {
+                    RGB_SPI,
+        RGB_RMOD,               RGB_MOD,
+                    RGB_SPD
+    },
+    [_PM_GAME] = {
+                    KC_W,
+        KC_A,               KC_D,
+                    KC_S
+    }
+};
+
+```
+!> use `KC_NO` when no keycode is desired, use of `KC_TRNS` or `_______` is unsupported as these maps do not act like layers _(only one can be active per pointing device at a time)_.
+
+The mode map array starts at index 0 and **must** be in the **same order** as the mode_ids of the maps (i.e `<mode map array index> + POINTING_MODES_MAP_START = <mode map mode_id>`), so use of `POINTING_MODES_MAP_START` is essential. However you can have more modes than you have maps as not all modes will be based on key presses (_such as advanced mode using `pointing_modes_task_*` functions_).
+
+### 8-Way Mode maps
+
+If `POINTING_MODES_8WAY_MAP_ENABLE` is defined then mode maps expect 8 keycodes per map but will allow having unique keycodes for diagonal directions as long as the mode type is set to `PMT_8WAY` (`PMT_4WAY` will still be the default). All maps **must** have 8 keycodes even if it is intended to use only the cardinal directions (`PMT_4WAY` or `PMT_DPAD`), it is recommended to fill unused directions with `KC_NO` as this helps with clarity.
+
+```c
+// Pointing Device Mode Maps 8-Way Format:
+const uint16_t PROGMEM pointing_modes_maps[][POINTING_MODE_NUM_DIRECTIONS] = {
+    [0] = {
+        <first map keycode up left>,   <first map keycode up>,    <first map keycode up right>,
+        <first map keycode left>,                                 <first map keycode right>,
+        <first map keycode down left>, <first map  keycode down>, <first map keycode down right>
+    },
+    //... all other pointing mode maps ...
+    [<POINTING_MODE_MAP_COUNT - 1>] = {
```

Comment:
`POINTING_MODE_MAP_COUNT` doesn't exist in the latest iteration of the code, looks like a leftover from a previous iteration?